docs/_extra/api-reference/hypothesis-v1.yaml

```diff
@@ -851,76 +877,136 @@ paths:
   # ---------------------------------------------------------------------------
 
   # ---------------------------------------------------------------------------
-  # GET groups/{id}/members - Get group members
+  # GET groups/{id}/members - Get group memberships
   # ---------------------------------------------------------------------------
   /groups/{id}/members:
     get:
       tags:
-        - groups
-      summary: Get group members
+        - memberships
+      summary: Get group memberships
       description: |
-        Fetch a list of all members (users) in a group. Returned user resource only
-        contains public-facing user data. Authenticated user must have read access
-        to the group. Does not require authentication for reading members of
-        public groups. Returned members are unsorted.
+        Get a paginated list of all user memberships in a group.
+        To get the memberships of a private group the authenticated user must be a member of the group.
+        Getting the memberships of an open or restricted group does not require authentication.
+        The returned memberships are sorted by joining date, oldest first.
+
+        If the `page[number]` query param is included in the request then the response will be paginated.
+        If there's no `page[number]` query param in the request then a legacy,
+        un-paginated response format will be used. In the future this legacy
+        un-paginated response format will be removed and the paginated response
+        format will be used even when the `page[number]` query param is not
+        present.
+      parameters:
+        - $ref: '#/components/parameters/PageNumber'
+        - $ref: '#/components/parameters/PageSize'
       security:
         - AuthClient: []
         - ApiKey: []
-        - {} # Unauthenticated OK for public groups
+        - {}
       responses:
         '200':
           description: Success
           content:
             application/*json:
               schema:
-                type: array
-                items:
-                  $ref: '#/components/schemas/User'
+                oneOf:
+                  - title: "Paginated"
+                    type: object
+                    properties:
+                      meta:
+                        description: "Metadata about this response."
+                        type: object
+                        properties:
+                          page:
+                            $ref: '#/components/schemas/PaginationMeta'
+                      data:
+                        description: "The list of memberships for the requested page."
+                        type: array
+                        items:
+                          $ref: '#/components/schemas/Membership'
+                  - title: "Unpaginated (deprecated)"
```

Comment:
If this is deprecated, I reckon you can also add `deprecated: true`, but I would need to check the docs to confirm this in fact possible in this context.

Reply:
I actually did try this but it didn't seem to do anything